common.conf

```diff
@@ -225,6 +221,8 @@ build += {
     //[info] [error]  ClassNotFoundException: : scala.util.Left  (Formats.scala:223)
     // !!!: probably we should investigate
     extra.run-tests: false
+    // with 0.13.13-RC1 I hit https://github.com/sbt/sbt/pull/2749
+    extra.sbt-version: "0.13.12"
```

Comment:
ha, well, it didn't actually work, it still extracted with 0.13.13-RC1, I guess `sbt-version` only controls the version used during building